tests/dotnet/BundleStructure/shared.csproj

```diff
@@ -103,6 +107,10 @@
 			<!-- Bundled, linked with, install_name_tool must have been executed -->
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 		</None>
+		<None Include="$(RootTestsDirectory)/test-libraries/libraries/.libs/$(RuntimeIdentifier)/libSkipInstallNameTool.so">
+			<!-- Bundled, not linked with, install_name_tool must have been executed -->
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</None>
 	</ItemGroup>
```

Comment:
The NoneQ.so file you added doesn't seem to be used anywhere, so let's try to do that:

```suggestion
		<None Update="NoneQ.so">
			<!-- Bundled, not linked with -->
			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
		</None>
	</ItemGroup>
```

this will require changes in the BundleStructureTest.cs file as well.

Reply:
I tried replicating the same structure as NoneQ, that have `NoneQ.dylib` and `libNoneQ.dylib`, so I created `NoneQ.so` and `libNoneQ.so`.  It seems that `NoneQ.dylib` is not checked either and that's the reason why I simply added `NoneQ.so`. I though they just need to be there and simply ignored, since they are not real libraries, it's just a text file with the file extension.